install_instructions/client/raspberry-pi.md

```diff
@@ -1,67 +1,71 @@
 ---
-lastUpdated: "2021-04-06 12:11:00"
+lastUpdated: "2021-05-03 12:25:00"
 streamPiVersion: "1.0.0"
-editedBy: "SamuelQuinones"
+editedBy: "rnayabed, SamuelQuinones"
 ---
 
 # Setup Stream-Pi on Raspberry-Pi
 
-The instructions below will go over how to set up a **Stream-Pi Client** on a **RaspberryPi** or a **Linux arm7** device.
+The instructions below will go over how to set up **Stream-Pi Client** on a **RaspberryPi** running **Raspberry Pi OS 32-Bit**. For a general Linux ARMv7 tutorial for other systems, follow the `Linux ARMv7` tutorial.
+
+**This method is NOT to be followed if your Raspberry Pi is running a 64 bit OS.**. Linux aarch64 builds are in the works, and will be available from the next release.
 
 If you have any questions please reach out to us on [the Official Stream-Pi Discord Server](https://discord.gg/BExqGmk).
 
-On some arm7 systems, specifically the RaspberryPi 3 you may experience a where your icon pictures will not render. This only seems to happen when the device is in console mode. Ensure your device is getting the appropriate power it needs and usually this can fix it. In the event that it does not, you can use use Desktop/GUI mode.
+Unlike Server, Stream-Pi Client has two main methods of running on Raspberry Pi and other ARMv7 devices. Desktop Mode and Console Mode.
 
 ## Desktop vs Console Mode
 
-Generally, **_we recommend console mode_** as Desktop mode may result in slower / poorer performance.
+Desktop Mode is to be used when you want to use Stream-Pi as a standard app with its window that you can resize and adjust. 
+
+Console Mode is to be used when you want a clean look without any windowing system.
+
+Generally, **_we recommend console mode_** as Desktop mode may result in slower / poorer performance due to lack of hardware acceleration.
+
+
+**There is a bug in the latest release which prevents some ARMv7 devices like the Raspberry Pi 3 (Raspberry Pi OS) to render icons while running Stream-Pi in console mode. The only solution then is to run Stream-Pi Client in Desktop mode. This issue has been already patched and will no longer be there in future releases.**
 
-## Client Instructions
+
+## Instructions
```

Comment:
"Client Instructions" was a conscious choice to eliminate confusion for end users, please undo this change